src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -75,11 +75,23 @@ default String getAsTextHidePrivate() {
         }
         if (!getAddress().isPrivate()) {
             builder.append(" Address: ").append(getAddress());
-        }
+        }*/
```

Comment:
Try and not leave commented out code in your file. You can use the Version Control Software to retrieve past codes.